va/va.go

```diff
@@ -650,6 +650,9 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, req *v
 			// `processRemoteResults` set the challenge status to valid so the
 			// validationTime metrics increment has the correct result label.
 			challenge.Status = core.StatusValid
+			// Timestamp the valid challenge.
+			currentTime := time.Now().UTC()
```

Comment:
This, and the other two spots. should use `va.clk.Now()` instead of `time.Now()`, so that the time they give can be controlled by the fakeClock supplied by tests.

Reply:
Per some of the other reviews I changed from getting currentTime to using an existing time we took at the beginning of the validation which uses va.clk.Now().